tests/test_reporter.py

```diff
@@ -1,7 +1,44 @@
-# from unittest.mock import patch
-#
-# from dacsspace.reporter import CSVReporter
+import os
+import unittest
+from unittest.mock import patch
```

Comment:
I think these imports could probably be improved a bit. I'm not convinced there need to be two lines, but if there do it could be more like:

```suggestion
from unittest import TestCase
from unittest.mock import patch
```

Reply:
I tried running tox with those lines but it returned an error that unittest was not defined

Reply:
Try running `flake8` first--that'll give you a more specific line-by-line thing about what's happening, but I think you'll need to change `unittest.TestCase` to `TestCase` since you're changing how it's imported.

Reply:
that did it! thanks!